### DISCUSSION TITLE
Timeseries panel option for use the average of the series in the y-axis

### DISCUSSION THREAD

**Discussion by lucamir** [grafana/grafana#57536] [Dashboards, Panels & Visualizations]
It would be nice to have a switch to set the average of the time series + a value as the min and the same thing for the max value.
In this way, the series is always kept within a certain range and not used the peaks of the series as range of the minimum and maximum value for the y-axis.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
